grapple/types/images.py

```diff
@@ -72,6 +77,7 @@ class ImageObjectType(DjangoObjectType, BaseImageObjectType):
         jpegquality=graphene.Int(),
     )
     src_set = graphene.String(sizes=graphene.List(graphene.Int))
+    tags = graphene.List(graphene.String)
```

Comment:
Not sure if we want to handle the tags on this PR.
Since tags are used on other models, they should have the same logic.
@NathHorrigan @zerolab what do you think of adding `tags` fields and resolvers on a new PR?

Reply:
This was not meant to be committed. This was a WIP for another PR...